src/cronjobs.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import {CronJob} from 'cron'
+import Offer from './models/Offer'
+import forms from './models/forms'
+import UserAction, {
+  ACTION_AUTOMATICALLY_DELETED_EXPIRED,
+  ACTION_AUTOMATICALLY_DELETED_NOT_CONFIRMED
+} from './models/UserAction'
+
+const TIMEZONE = 'Europe/Berlin'
+
+const deleteOffer = async (offer: Offer) => {
+  const {FormModel} = forms[offer.city]
+  // Delete the form data
+  await FormModel.findByIdAndDelete(offer.formData._id).exec()
+  // Delete the offer
+  await Offer.findByIdAndDelete(offer._id).exec()
+}
+
+const deleteExpired = async () => {
+  const twoYearsAgo = new Date()
+  twoYearsAgo.setFullYear(twoYearsAgo.getFullYear() - 2)
```

Comment:
We have the moment dependency in the project. It has a fluent api like: `moment('2013-04-01T00:00:00.000').subtract(1, 'ms')`

I think we should stick with it and use it everywehre

Reply:
Done. While doing it I changed every use of Date to moment.